docs.yml

```diff
@@ -2,5 +2,5 @@ title: over_react
 base: github:Workiva/over_react/
 src: README.md
 topics:
-  - title: Transformer
-    src: lib/src/transformer/README.md
+  - title: Builder
+    src: lib/src/builder/README.md
```

Comment:
Should we add the snippets readme here so they can be searched for / found in the dev portal?

Reply:
yup, good catch!